README.md

```diff
@@ -1,27 +1,38 @@
 
 ## Jekyll Installation
-Swap out `rbenv` below for `rvm` if you prefer. RVM was giving me installation issues, so I found `rbenv` - Travis
+Jekyll runs on Ruby, and running multiple versions of Ruby on a single system is apparently difficult enough to support at least two environment managers, `rbenv` and `rvm`, which allow you to select which version of Ruby you'd like to use. We include instructions here for `rbenv`, but feel free to use `rvm` if you'd like.
 
-1. Create and run from a fresh VM instance:
-    1. `gcloud compute instances create jekyll --image-project=ubuntu-os-cloud --image-family=ubuntu-1404-lts --machine-type=n1-standard-1`
+1. (Optional) Create and run from a fresh VM instance:
```

Comment:
I think it might be worthwhile to mention the alternative, installing on a local machine. Also, if I install on a vm, I have to forward 4000, right?

Reply:
Correct - Step 2 includes the ssh flag to do that. Will add text to that effect.